.github/labeler.yml

```diff
@@ -9,6 +9,18 @@ labels:
   files:
     - "docs/.*"
 
+- label: "tests"
+  files:
+    - "tests/.*"
+
+- label: "CMake"
+  files:
+    - ".*CMakeLists.txt"
+
+- label: "CMake"
+  files:
+    - "cmake/.*"
```

Comment:
No need to add the label twice?
```suggestion
- label: "CMake"
  files:
    - ".*CMakeLists.txt"
    - "cmake/.*"
```

Reply:
No, the files must match *all* listed regular expressions, so it's an implicit conjunction. For disjunctions, we need to repeat the rule.

Reply:
Wait, this is not the official GitHub labeler? I'm looking at the wrong documentation.

Reply:
See https://github.com/srvaroa/labeler#configuration:

> The action combines all conditions with an AND operation.
> ...
> If you wish to apply an OR, you may set multiple matchers for the same label.

Reply:
Right, it's not the official one. This one has more features and is actively developed.

Reply:
I'm looking at:
`- uses: actions/labeler@v4`
https://github.com/marketplace/actions/labeler

But you're using:
`- uses: srvaroa/labeler@master`